README.MichalAW.md

```diff
@@ -0,0 +1,20 @@
+# WDP190504-11
```

Comment:
po co ten plik w projekcie?

Reply:
@MichalAW , zgodnie z rozmową - dokumentacja miala sie tyczyć jakichś konkretnych plików js/html z kilkuzdaniowym opisem typu "This code is responsible for ...", nie do szczegółowego opisu zmian w danym PRze. 

Tego typu komentarze można dodać do opisu samego PRa ale nie jako osobny plik w repo.